packages/components/src/components/Forms/FormText.tsx

```diff
@@ -0,0 +1,167 @@
+import { Text } from 'react-native'
+import React, { FC } from 'react'
+
+import { StringOrTextWithA11y } from '../../types/common'
+import { getA11yText, getDisplayText } from '../../utils'
+import { useTheme } from '../../utils'
+import { useTranslation } from 'react-i18next'
```

Comment:
Library/local import mixing.

Wonder if we can update our eslint or TS config to handle this for us--I feel like whatever configuration flagship has does automatically handle split sorting libraries then local.

Reply:
Fixed